# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/chrome-webidl/WindowGlobalActors.webidl
# Commit: 13a9d145a550
# Full Hash: 13a9d145a55053585dd7477bdc8b7842606784a6
# Author: Toshihito Kikuchi <tkikuchi@mozilla.com>
# Date: 2021-12-17 03:51:53
# Description:
#   Bug 1743663 - Remove windowContext.hasActivePeerConnections.  r=peterv
#   
#   TabUnloader.jsm can directly call windowGlobal.hasActivePeerConnections(),
#   so windowContext.hasActivePeerConnections is not needed.
#   
# ==============================================================================

diff -r 630013ad7f37 -r 13a9d145a550 dom/chrome-webidl/WindowGlobalActors.webidl
--- a/dom/chrome-webidl/WindowGlobalActors.webidl	Thu Dec 16 15:44:15 2021 +0000
+++ b/dom/chrome-webidl/WindowGlobalActors.webidl	Thu Dec 16 16:14:13 2021 +0000
@@ -33,11 +33,6 @@
   // It won't become false if the image becomes non-lazy.
   readonly attribute boolean hadLazyLoadImage;
 
-  // True if any of the windows in the subtree rooted at this window
-  // has active peer connections.  If this is called for a non-top-level
-  // context, it always returns false.
-  readonly attribute boolean hasActivePeerConnections;
-
   /**
    * Partially determines whether script execution is allowed in this
    * BrowsingContext. Script execution will be permitted only if this
@@ -162,6 +157,11 @@
    */
   [Throws]
   Promise<nsITransportSecurityInfo> getSecurityInfo();
+
+  // True if any of the windows in the subtree rooted at this window
+  // has active peer connections.  If this is called for a non-top-level
+  // context, it always returns false.
+  boolean hasActivePeerConnections();
 };
 
 [Exposed=Window, ChromeOnly]